components/data_comps/datm/cime_config/config_component.xml

```diff
@@ -54,6 +54,7 @@
       <value compset="^RCP4_">rcp4.5</value>
       <value compset="^RCP2_">rcp2.6</value>
       <value compset="^HIST_">trans_1850-2000</value>
+      <value compset="^20TR_">trans_1850-2000</value>
```

Comment:
This comment really applies to all changes in the data components config_componetn.xml files
- rather than having two lines that are repeated - it would be cleaner to have and entry of the form 
<value compset="^HIST_|^20TR_">trans_1850-2000</value>
However, I think this needs to be done in a separate PR to clean this up. Once this PR is accepted can we make sure to open a separate issue to cleanup duplicates like this?